package.json

```diff
@@ -77,8 +77,8 @@
     "ui-select": "^0.19.6"
   },
   "devDependencies": {
-    "@spinnaker/amazon": "^0.0.173",
-    "@spinnaker/core": "0.0.362",
```

Comment:
wowza

Reply:
i know. it hurts. there are more commits to this PR coming because apparently `react` and all its friends need updating to make CI pass (all the new hooks stuff blows up when karma tries to build/use it)